pallets/parachain-staking/src/lib.rs

```diff
@@ -716,6 +727,98 @@ pub mod pallet {
 			Ok(())
 		}
 
+		/// Increases the maximum amount a collator candidate can stake. The new
+		/// maximum cannot be lower than MinCollatorCandidateStk.
+		///
+		/// The dispatch origin must be Root.
+		///
+		/// Emits `MaxCandidateStakeChanged`.
+		///
+		/// # <weight>
+		/// Weight: O(1)
+		/// - Reads: [Origin Account], MaxCollatorCandidateStk
+		/// - Writes: Round
+		/// # </weight>
+		#[pallet::weight(100_000)]
+		pub fn increase_max_candidate_stake(origin: OriginFor<T>, new: BalanceOf<T>) -> DispatchResult {
+			frame_system::ensure_root(origin)?;
+			ensure!(new >= T::MinCollatorCandidateStk::get(), Error::<T>::CannotSetBelowMin);
+			let old = <MaxCollatorCandidateStk<T>>::get();
+			ensure!(new > old, Error::<T>::NotIncreasing);
+			<MaxCollatorCandidateStk<T>>::put(new);
+
+			Self::deposit_event(Event::MaxCandidateStakeChanged(old, new));
+			Ok(())
+		}
+
+		/// Increases the maximum amount a collator candidate can stake. The new
+		/// maximum cannot be lower than MinCollatorCandidateStk.
+		///
+		/// The dispatch origin must be Root.
+		///
+		/// Emits `MaxCandidateStakeChanged`.
+		///
+		/// # <weight>
+		/// - The transaction's complexity is mainly dependent on updating the
+		///   `SelectedCandidates` storage in `select_top_candidates` which in
+		///   return depends on the number of `MaxSelectedCandidates` (N).
+		/// - For each N, we read `CollatorState` from the storage.
+		/// ---------
+		/// Weight: O(N) where N is `MaxSelectedCandidates` bounded by
+		/// `MaxCollatorCandidates`
+		/// - Reads: MaxCollatorCandidateStk, 2 * N * CollatorState,
+		///   CandidatePool
+		/// - Writes: MaxCollatorCandidateStk, N * CollatorState,
+		///   SelectedCandidates
+		/// # </weight>
+		#[pallet::weight(100_000)]
+		pub fn decrease_max_candidate_stake(origin: OriginFor<T>, new: BalanceOf<T>) -> DispatchResultWithPostInfo {
+			frame_system::ensure_root(origin)?;
+			ensure!(new >= T::MinCollatorCandidateStk::get(), Error::<T>::CannotSetBelowMin);
+
+			let old = <MaxCollatorCandidateStk<T>>::get();
+			ensure!(old > new, Error::<T>::NotDecreasing);
+
+			// iterate candidate pool and update all candidates with stake > max
+			let mut post_weight = T::DbWeight::get().reads(2);
+			<CandidatePool<T>>::mutate(|pool| {
```

Comment:
You should start using bounded vec stuff here or you will need to migrate later

Reply:
Will do in a separate PR. Thanks for pointing that out!